packages/suite/src/views/wallet/send/components/Outputs/components/Amount/index.tsx

```diff
@@ -91,33 +103,152 @@ const Amount = ({ output, outputId }: Props) => {
         setMax,
         calculateFiat,
         composeTransaction,
+        trigger,
     } = useSendFormContext();
 
     const theme = useTheme();
+    const { areSatsDisplayed, isSupportedByCurrentNetwork } = useBitcoinAmountUnit();
+    const areSatsUsed = areSatsDisplayed && isSupportedByCurrentNetwork;
+
+    useDidUpdate(() => {
+        trigger();
```

Comment:
possible race condition
im not sure that it will not break something. take a note that useSendForm > useFormDraft is called after mount.
loading form from draft is an async process (mount > TrezorConnect.compose > processing data > filling the form)

Reply:
Not necessary, removed